### ISSUE TITLE
Experimentation with Flux tiling

### ISSUE THREAD

**Issue by thipokKub** [spinagon/ComfyUI-seamless-tiling#14]
Hello, I've been experimenting with tiling sampling as I want to produce seamless panoramic image using flux. But it seems like the current method is to shift the image, and do in-painting. But I think there is another way, so I implement a simple euler sampling with minor modification by padding the input sample left side with right patch, and vice versa. On every step it then update the padded latent from the center piece

```python
@torch.no_grad()
def sample_euler_wraparound(model, x, sigmas, extra_args=None, callback=None, disable=None, s_churn=0., s_tmin=0., s_tmax=float('inf'), s_noise=1.):
    """Implements Algorithm 2 (Euler steps) from Karras et al. (2022). Modified to wrap around on the y-axis."""
    extra_args = {} if extra_args is None else extra_args
    s_in = x.new_ones([x.shape[0]])
    pad_width = max(min(int(x.shape[3] * 0.25), 32), 4)
    for i in trange(len(sigmas) - 1, disable=disable):
        x_padded = torch.cat([x[:, :, :, -pad_width:], x, x[:, :, :, :pad_width]], dim=3)

        if s_churn > 0:
            gamma = min(s_churn / (len(sigmas) - 1), 2 ** 0.5 - 1) if s_tmin <= sigmas[i] <= s_tmax else 0.
            sigma_hat = sigmas[i] * (gamma + 1)
        else:
            gamma = 0
            sigma_hat = sigmas[i]

        if gamma > 0:
            eps = torch.randn_like(x) * s_noise
            x_padded = x_padded + eps * (sigma_hat ** 2 - sigmas[i] ** 2) ** 0.5

        denoised = model(x_padded, sigma_hat * s_in, **extra_args)
        d = to_d(x_padded, sigma_hat, denoised)

        x = x_padded[:, :, :, pad_width:-pad_width]

        if callback is not None:
            callback({'x': x, 'i': i, 'sigma': sigmas[i], 'sigma_hat': sigma_hat, 'denoised': denoised})
        dt = sigmas[i + 1] - sigma_hat
        # Euler method
        x = x + d[:, :, :, pad_width:-pad_width] * dt
  
    return x
```

This works out okay, and achieve somewhat blended seam (but seams can still be seen). But it was usable when apply with panoramic lora

## Example
I generate the image of size 768 x 512, and stitch them in photoshop to show the blending

Prompt
`
A photograph of a bedroom in futuristic interior style. The room consist of a single bed, a door, and large windows opening to garden
`

Without lora
![Image](https://github.com/user-attachments/assets/19896b75-8861-4bfa-8679-a44722914dd9)

With lora
![Image](https://github.com/user-attachments/assets/101677f7-9c53-448c-849f-d3f19d4ac6c0)

## Limitation

I tried to apply the controlnet, but the padded latent seems to be govern by controlnet, and is cropped out in the last part. To mitigate this, I need to manually wrap around the controlnet image
